mifospay/src/main/res/layout/fragment_si.xml

```diff
@@ -20,29 +19,29 @@
         android:scrollbars="vertical" />
 
     <include
-        android:visibility="gone"
-        android:id="@+id/inc_state_view"
+        android:id="@+id/inc_state_view_1"
```

Comment:
You can simply remove this one. We don't need two layouts for the same thing

Reply:
done